test-framework/arquillian/src/main/java/io/quarkus/arquillian/QuarkusDeployableContainer.java

```diff
@@ -195,6 +195,7 @@ public void execute(BuildContext context) {
             Thread.currentThread().setContextClassLoader(runningQuarkusApplication.getClassLoader());
             // Instantiate the real test instance
             testInstance = TestInstantiator.instantiateTest(testJavaClass, runningQuarkusApplication.getClassLoader());
+            System.err.println("Setting testInstance to " + testInstance);
```

Comment:
@FroMage forgot to erase debugging evidence? ;-)

Reply:
DAMNIT

Reply:
OK should be fixed, thanks a lot :)